Sources/Controllers/VolumeController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  VolumeController.swift
+//  ControlKit
+//
+
+import Control
+import SwiftUI
+
+/// A controller for managing audio volume and mute state.
+///
+/// `VolumeController` provides functionality to control the audio volume and mute state of the device.
+///
+@MainActor
+public final class VolumeController: ObservableObject {
+    
+    @Published public var volume: Float = Control.Volume.volume
```

Comment:
It appears that this property can also be marked as `private(set)` since there is no subscription to changes from outside sources? 👀